game.js

```diff
@@ -86,10 +124,11 @@ const clicks = {
 		});
 		$('.tile').on('contextmenu', function(e){
 		return false;
-		});
+	}); // WTF??
```

Comment:
disables context menu on right click
so that when you right click a tile, it just places a flag, instead of placing a flag and opening the context menu